pkg/handlers/v1/dependencycheck.go

```diff
@@ -0,0 +1,19 @@
+package v1
+
+import (
+	"context"
+
+	"github.com/asecurityteam/ipam-facade/pkg/domain"
+)
+
+// DependencyCheckHandler takes in a domain.DependencyChecker to check external dependencies
+type DependencyCheckHandler struct {
+	DependencyChecker domain.DependencyCheck
+}
+
+// Handle makes a call CheckDependencies from DependencyChecker that verifies this
+// app can talk to it's external dependencies
+func (h *DependencyCheckHandler) Handle(ctx context.Context) error {
+
```

Comment:
nit: no need for the empty line here. 🤷‍♂

Reply:
changed